tldextract/tldextract.py

```diff
@@ -187,7 +187,8 @@ def __call__(self, url):
             .partition("#")[0] \
             .split("@")[-1] \
             .partition(":")[0] \
-            .rstrip(".")
+            .rstrip(".") \
+            .strip()
```

Comment:
Should this go before the `rstrip` call? The trailing dot the `rstrip` is meant to take care of would not be stripped if there were whitespace in its way.